verification/curator-service/ui/src/components/CaseForm.tsx

```diff
@@ -458,338 +483,459 @@ class CaseForm extends React.Component<Props, CaseFormState> {
-                                        ['notes'],
-                                        errors,
-                                        touched,
-                                    ),
-                                })}
-                                Notes
-                            </div>
-                        </nav>
-                        <div className={classes.form}>
-                            <Form>
-                                <div className={classes.formSection}>
-                                    <Source
-                                        initialValue={values.caseReference}
-                                    ></Source>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <Demographics></Demographics>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <LocationForm></LocationForm>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <Events></Events>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <Symptoms></Symptoms>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <PreexistingConditions></PreexistingConditions>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <Transmission></Transmission>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <TravelHistory></TravelHistory>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <GenomeSequences></GenomeSequences>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <Pathogens></Pathogens>
-                                </div>
-                                <div className={classes.formSection}>
-                                    <Notes></Notes>
-                                </div>
-                                {isSubmitting && <LinearProgress />}
-                                <br />
-                                <Button
-                                    variant="contained"
-                                    color="primary"
-                                    data-testid="submit"
-                                    disabled={isSubmitting}
-                                    onClick={submitForm}
+            <>
+                <Modal open={true}>
```

Comment:
The bulk case form is also mocked as a modal.

Is there any way we can share the modal logic as a discrete component that contains the contents of either form? Or partially share it?

Reply:
Definitely worth considering. I kept the bulk case change out of this PR since it was already pretty big. Let's keep this in mind when we change the bulk case to a modal.